src/popup/router/components/TabBar.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+  <div class="tab-bar">
+    <button :class="{ selected: $route.path === '/account' }">
+      <router-link to="/account">
+        <Home />
+      </router-link>
+    </button>
+    <button>
+      <router-link to="/">
+        <Balances />
+      </router-link>
+    </button>
+    <button>
+      <router-link to="/">
+        <Payments />
+      </router-link>
+    </button>
+    <button :class="{ selected: $route.path === '/tip' }">
+      <router-link to="/tip">
+        <Tips />
+      </router-link>
+    </button>
+    <button :class="{ selected: $route.path === '/settings' }">
+      <router-link to="/settings">
+        <Settings />
+      </router-link>
+    </button>
+  </div>
+</template>
+
+<script>
+import Home from '../../../icons/sidebar-menu/home.svg?vue-component';
+import Balances from '../../../icons/sidebar-menu/balances.svg?vue-component';
+import Payments from '../../../icons/sidebar-menu/payments.svg?vue-component';
+import Tips from '../../../icons/sidebar-menu/tips.svg?vue-component';
+import Settings from '../../../icons/settings.svg?vue-component';
+
+export default {
+  components: { Home, Balances, Payments, Tips, Settings },
+};
+</script>
+
+<style lang="scss" scoped>
+@import '../../../styles/variables';
+
+.tab-bar {
+  display: flex;
+  background-color: $color-bg-3;
+  border-top: 1px solid $color-bg-3;
+
+  > button {
+    display: flex;
+    padding: 0;
+    border-radius: 2px 2px 0 0;
+    background-color: $color-bg-2;
+    border-right: 1px solid $color-bg-3;
+    border-left: 1px solid $color-bg-3;
+
+    &.selected {
+      background-color: $color-bg-1;
+
+      a svg {
+        color: $color-blue;
+      }
```

Comment:
```suggestion
      color: $color-blue;
```
won't it work the same?